debian/jitsi-meet-prosody.postinst

```diff
@@ -132,7 +132,7 @@ case "$1" in
 
         # Check whether prosody config has the internal muc, if not add it,
         # as we are migrating configs
-        if ! grep -q "internal.auth.$JVB_HOSTNAME" $PROSODY_HOST_CONFIG; then
+        if [ -f $PROSODY_HOST_CONFIG ] && ! grep -q "internal.auth.$JVB_HOSTNAME" $PROSODY_HOST_CONFIG; then
```

Comment:
I think you want to quote the string "$PROSODY_HOST_CONFIG" in case there's spaces or anything else.